src/sentry/models/integration.py

```diff
@@ -13,19 +13,35 @@
 from sentry.signals import integration_added
 
 
-class PagerDutyServiceProject(Model):
+class PagerDutyService(Model):
     __core__ = False
 
-    project = FlexibleForeignKey("sentry.Project", db_index=False, db_constraint=False)
     organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration")
     integration_key = models.CharField(max_length=255)
     service_id = models.CharField(max_length=255)
     service_name = models.CharField(max_length=255)
 
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_pagerdutyservice"
+        unique_together = (("service_id", "integration_key", "organization_integration"),)
```

Comment:
Still don’t get why the integration_key is part of the unique_together. If the integration_key is changed, shouldn’t we overwrite the existing entry?

Reply:
although I don't know how we'd even get the new integration key info, you can re-generate them so I'll take that out